docs/reference/autoscaling/apis/get-autoscaling-capacity.asciidoc

```diff
@@ -238,6 +238,268 @@ The API returns the following result:
 [source,console-result]
 --------------------------------------------------
 {
-  policies: {}
+  "policies" : {
```

Comment:
These responses contain a lot of details, I wonder if we should provide more information about what the different sections mean, but since this API is meant to be used internally I'm not sure...